### PR TITLE
chore: LP tooltips & move market-info lib

### LINKED ISSUE #1282: Market info library

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #1282 

# Technical 👨‍🔧 & Description ℹ️

- [x] refactor market info to be its own lib
- [x] update routing - from the liquidity page you should be able to click on trading and navigate back to the last selected market 
- [x] add LP tooltips (added together with @candida-d)
- [x] fixing market-info overflow in auto-sizer panels
- [x] fix network switcher on a lighter background 

# Demo 📺

Screenshot 2022-09-09 at 14 38 43
Screenshot 2022-09-09 at 14 38 28
Screenshot 2022-09-09 at 14 37 44

### CODE REVIEW

File: libs/market-info/src/components/MarketNames.graphql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+query MarketNames {
+  markets {
+    id
+    name
```
Comment (MadalinaRaicu) [inline libs/market-info/src/components/MarketNames.graphql:4]: `name` should be `tradableInstrument.instrument.name`

File: libs/ui-toolkit/src/components/tooltip/tooltip.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import {
   Arrow,
   Portal,
 } from '@radix-ui/react-tooltip';
+import type { ITooltipParams } from 'ag-grid-community';
```
Comment (asiaznik) [inline libs/ui-toolkit/src/components/tooltip/tooltip.tsx:11]:
```suggestion
import type { ITooltipParams as AgTooltipProps } from 'ag-grid-community';
```

File: libs/ui-toolkit/src/components/tooltip/tooltip.tsx [hunk 3]
```diff
@@ -56,3 +57,11 @@ export const Tooltip = ({
   ) : (
     children
   );
+
+export const TooltipCellComponent = (props: ITooltipParams) => {
```
[the remark this answers is not in the document]
Comment (MadalinaRaicu) [reply libs/ui-toolkit/src/components/tooltip/tooltip.tsx:61]: I have nothing against this. But, I will need to change all cell renderer names to follow this naming convention, for consistency.

[the remark this answers is not in the document]
Comment (MadalinaRaicu) [reply libs/ui-toolkit/src/components/tooltip/tooltip.tsx:61]: https://github.com/vegaprotocol/frontend-monorepo/issues/1299 we'll address this here

File: libs/ui-toolkit/src/components/tooltip/tooltip.tsx (hunk 2 above)
Comment (MadalinaRaicu) [reply libs/ui-toolkit/src/components/tooltip/tooltip.tsx:11]: https://github.com/vegaprotocol/frontend-monorepo/issues/1299 we'll address this here

File: libs/market-info/src/components/market-info/info-market.tsx [hunk 4]
```diff
@@ -218,16 +226,6 @@ export const Info = ({ market }: InfoProps) => {
         />
       ),
     },
-    {
-      title: t('Risk factors'),
```
Comment (notbot00) [inline libs/market-info/src/components/market-info/info-market.tsx:222]: question (??): i don't have any context on this, but is there a reason we don't want to display this anymore?

**Review by notbot00 — APPROVED** [review]

File: libs/market-info/src/components/market-info/info-market.tsx (hunk 4 above)
Comment (MadalinaRaicu) [reply libs/market-info/src/components/market-info/info-market.tsx:222]: riskFactors are missing in some of the market queries. happy to add this back once things get sorted. in the meantime  I want to remove this if it breaks the whole market-info view. the console will be used for fairground incentives and I don't want to leave this breaking bit here. 

image

Comment (MadalinaRaicu) [reply libs/market-info/src/components/market-info/info-market.tsx:222]: IMO the best approach would be to add riskFactors back after we update the market data providers.

**Pull request merged**